src/main/java/com/influxdb/v3/client/config/ClientConfig.java

```diff
@@ -96,9 +101,16 @@ public final class ClientConfig {
     private final Duration timeout;
     private final Boolean allowHttpRedirects;
     private final Boolean disableServerCertificateValidation;
-    private final ProxySelector proxy;
+    private final InetSocketAddress proxyAddress;
```

Comment:
I would prefer a String type here if possible, so that users don't also have to add `import java.net.InetSocketAddress;`. From the library point of view`InetSocketAddress` can be handled internally using values from the parsed proxy string.  

Also can this be added to the unit tests in https://github.com/InfluxCommunity/influxdb3-java/blob/main/src/test/java/com/influxdb/v3/client/config/ClientConfigTest.java